slack-request-verifier/src/main/java/com/hubspot/slack/client/request/verifier/SlackRequestVerifier.java

```diff
@@ -0,0 +1,11 @@
+package com.hubspot.slack.client.request.verifier;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+import com.google.inject.BindingAnnotation;
+
+@BindingAnnotation
+@Retention(RetentionPolicy.RUNTIME)
+public @interface SlackRequestVerifier {
```

Comment:
I feel like `SlackRequestVerifier` is a good name for the filter implementation but the marker's name could point to the action itself(e.g.`VerifySlackRequest`)?

Reply:
`VerifySlackRequest` sounds good to me too. Is it better to rename this annotation?